book/wildfires/sensors/wildfires-sensors-modis.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>add <code>import urllib.request</code> if we use <code>urllib.request.urlretrieve</code> to download MODIS wildfires, see comment in the line<code>! wget 'https://firms2.modaps.eosdis.nasa.gov/data/country/zips/modis_2020_all_countries.zip' -O 'modis_2020_all_countries.zip'</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>